pkg/ccl/backupccl/backup_job.go

```diff
@@ -156,6 +156,23 @@ func backup(
 		}
 	}
 
+	// Add the spans for any tables that are excluded from backup to the set of
+	// already-completed spans, as there is nothing to do for them.
+	descs := iterFactory.NewDescIter(ctx)
+	defer descs.Close()
+	for ; ; descs.Next() {
+		if ok, err := descs.Valid(); err != nil {
+			return roachpb.RowCount{}, 0, err
+		} else if !ok {
+			break
+		}
+
+		if tbl, _, _, _, _ := descpb.GetDescriptors(descs.Value()); tbl != nil && tbl.ExcludeDataFromBackup {
+			prefix := execCtx.ExecCfg().Codec.TablePrefix(uint32(tbl.ID))
+			completedSpans = append(completedSpans, roachpb.Span{Key: prefix, EndKey: prefix.PrefixEnd()})
```

Comment:
wait hold up: how should incremental backups respond after a user sets `exclude_data_from_backup ` back to false? I think we should treat them as NewSpans. But that would only happen if `backupManifest.Spans` for the _previous_ backup doesn't have these spans. I think this pr would introduce data corruption

Reply:
or maybe we already have a data corruption problem

Reply:
ok yeah, i think we have always had this problem
https://github.com/cockroachdb/cockroach/blob/35e82b3e894a60b1790dc22955ded67e8f96817b/pkg/ccl/backupccl/backup_planning.go#L1511

we bind this spans var to the backup manifest without considering if we exclude them from the backup

Reply:
approving this PR, this isn't a regression

Reply:
yeah, Aditya and I discussed this a bit in slack, and you can see my alteration to the test shows how we get a sorta weird number of rows based on the fact that we started excluding it in an inc layer, but I think it is working ~as intended.